docs/draft/3rd-party-api.md

```diff
@@ -0,0 +1,109 @@
+---
+title: 3rd party APIs used for core functionality that impacts things like availability/censorship and privacy
```

Comment:
Quite a mouthful...

I wonder if this shouldn't be separate specs based on _purpose_. E.g. "Status Transaction History Usage" or so, etc

cc @andremedeiros